packages/xo-web/src/common/editable/index.js

```diff
@@ -490,15 +490,26 @@ export class XoSelect extends Editable {
 
 export class Size extends Editable {
   static propTypes = {
-    value: PropTypes.number.isRequired,
+    value: PropTypes.oneOfType([
+      PropTypes.number.isRequired,
```

Comment:
Why do you keep the `isRequired` here?

Reply:
Because I forgot to remove it :innocent: